Makefile

```diff
@@ -23,92 +30,6 @@ IMAGE=us.gcr.io/stackrox-infra/infra-server:$(VERSION)
 image-name:
 	@echo $(IMAGE)
 
-#############
-## Linting ##
-#############
```

Comment:
Ugh. Re-organizing files makes review extra hard. Re-org _should_ be a separate PR IMhO. But not going to hold this PR back for it.